### PR TITLE
add some missing html configs; slight blur

### PR DESCRIPTION
**Pull request by SylvanSign**

### CODE REVIEW

File: public/styles/interface.css [hunk 1]
```diff
@@ -61,6 +61,7 @@ input:focus {
 
 .pulse {
   animation: pulse 3s linear infinite;
+  filter: blur(0.6px);
```
Comment (SylvanSign) [inline public/styles/interface.css:64]: makes the text slightly blurry, to add to the CRT look. lemme know if it's too much

File: public/index.html [hunk 2]
```diff
@@ -1,6 +1,10 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
 
 <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (SylvanSign) [inline public/index.html:6]: this is needed to make the page responsive eventually. Even now it helps make it look nicer on a mobile device

File: public/index.html [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
+  <title>Are you alive?</title>
```
Comment (SylvanSign) [inline public/index.html:7]: pending game title. This is the opening line to the first episode of BSG

Comment (ZakMiller) [reply public/index.html:7]: zzz

**Review by ZakMiller — APPROVED** [review]

**Pull request merged**